recipes/docker-py/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{%set name = "docker-py" %}
+{%set version = "1.8.1" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4f47a05e677472b5e022be1ab1dfd91b473ab3fc14a6b71337042ac2caaafa0b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+
+  run:
+    - python
+    - requests >=2.5.2
+    - six >=1.4.0
+    - websocket-client >=0.32.0
```

Comment:
For this release (though maybe not for later ones), it appears this is [exact]( https://github.com/docker/docker-py/blob/1.8.1/requirements.txt#L3 ).